src/zenml/integrations/mlflow/mlflow_utils.py

```diff
@@ -0,0 +1,159 @@
+#  Copyright (c) ZenML GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+import os
+from typing import Any, Callable, Optional, Type
+
+import mlflow  # type: ignore
+from mlflow import set_experiment
+from mlflow.tracking import set_tracking_uri  # type: ignore
+
+from zenml.io.utils import get_global_config_directory
+from zenml.pipelines.base_pipeline import BasePipeline
+from zenml.steps import BaseStep
+
+
+def local_mlflow_backend() -> str:
+    """Returns the local mlflow backend inside the global zenml directory"""
+    local_mlflow_backend_uri = os.path.join(
+        get_global_config_directory(), "mlruns"
+    )
```

Comment:
Storing the mlruns in the global config directory effectively means that I won't be able to share them with the local KFP orchestrator. Until we model the MLFlow tracker as a stack component, I think it would be better if you stored the mlruns in the root of the local artifact store (which is already shared with local KFP).

Reply:
This is something that we can do after, as a separate PR.